modules/installation-azure-user-infra-uploading-rhcos.adoc

```diff
@@ -58,17 +74,37 @@ less than or equal to the {product-title} version that you install. Use the imag
 that matches your {product-title} version if it is available.
 ====
 
-. Copy the chosen VHD to a blob:
+. Create the storage container for the VHD:
 +
 [source,terminal]
 ----
 $ az storage container create --name vhd --account-name ${CLUSTER_NAME}sa --account-key ${ACCOUNT_KEY}
 ----
+ifdef::ash[]
+. Download the compressed {op-system} VHD file locally:
++
+[source,terminal]
+----
+$ curl -O -L ${COMPRESSED_VHD_URL}
+----
+
+. Decompress the VHD file.
```

Comment:
We should note here that the uncompressed file is 16GB large so the host system must have 16GB of free space. They should feel free to delete image after it has been uploaded in step 7.